packages/cli/README.md

```diff
@@ -39,16 +38,16 @@ Run a local network of CLI Ultralight clients.  Test JSON-RPC calls in a termina
 - From `./packages/cli`:
 - Run `npm run devnet -- --numNodes=3`
   - This will start 3 nodes with JSON-RPC server addresses `[8545, 8546, 8547]`
-- To specify a port number to, include a `--port` variable
+- To specify a discv5 port number, include a `--port` variable
   - `npm run devnet -- --numNodes=5 --port=9009`
-  - This will start 5 nodes with JSON-RPC server addresses `[9009, 9010, 9011, 9012, 9013]`
+  - This will start 5 nodes with discv5 listener ports on `[9009, 9010, 9011, 9012, 9013]`
 - To specify which subnetworks to support, include one or more `--networks` parameters
-  - `npm run devnet -- --numNodes=5 --port=9009 --networks=history --networks=beacon`
+  - `npm run devnet -- --numNodes=5 --networks=history --networks=beacon`
```

Comment:
Is this really correct that this has to be passed by 2x providing the same option? 🤔 So not `--networks=history,beacon`?

Reply:
Lemme check if the other option works.  The way I've shown above is how `yargs` processes array params though as far as I know.